rfc/networked-dot-storage.md

```diff
@@ -0,0 +1,174 @@
+
+This is not the path we are going to pursue. It feels important to say that up front.
+
+But before we go deeper into where we are headed, it's worth looking at the implied dichotomy between "decentralization" and product features. That this dichotomy is almost reflexively referred to likely results from failing to appropriately frame "decentralization" in terms of product features. So let's start by framing the product features we want to unlock through decentralization.
+
+## The Network We Want
+
+### No Data Loss
+
+1. Users should be able to specify how many unique copies of their data they want backed up, and the W3S network should handle insuring that many copies remain backed up as long as the user pay to store in W3S.
+
+### Perpetuity
+
+1. Users should be able to store hot data in W3S that will be available for  retrieval until they stop paying for it. They should not have to worry about any single business entity holding their data disappearing. If it does, the data should go somewhere else without their needing to intervene.
+
+### Sensible defaults with customization
+
+1. Users should have the option to store with simply "the W3S network" rather than a specific business entity or Filecoin SP. This is a hard lesson learned from Filecoin that forcing users to choose from a marketplace where information on available options is hard to come by is a terrible user experience.
+
+2. Users should still have the option to choose a provider they like, or to move between providers.
+
+### Trust
+
+In general, users should be able to "trust" W3S to handle their data well and meet service level agreements. That trust should be derived by giving users the ability to verify any actions taken on the W3S network. 
+
+1. Users should be able to gather a detailed history of what has happened with their data when they put it on the W3S Network.
+
+2. A user paying for storage should be able to verify that data is being stored.
+
+3. A user paying for storage should be able to verify it can be retrieved quickly. 
+
+4. A user paying for multiple backups should be able to verify each backup is still functioning.
+
+5. If verification demonstrates that a particular provider is failing to provide a service that the user has paid for, the network should detect the failures and move their data automatically. (or alternatively, the user should have the ability to create a dispute and move to a different provider while getting money back)
+
+### Distribution of resources
+
+6. Users should not have to worry about large service providers establishing monopolies that reduce user choice or drive up costs. To insure this, W3S should emphasize data portabiltity, fairness in data distribution, and low barrier to entry for new providers.
```

Comment:
> data portabiltity

Love.

> low barrier to entry for new providers

Love!

> fairness in data distribution

This is the part I don't really get. Fairness in what sense?

Reply:
Essentially, for a user who doesn't want to make choices about their data, they should have assurance the network isn't betting only on large players, and is insuring they get the best service regardless of size. And, they should be able to verify that.